openmc/filter.py

```diff
@@ -626,106 +626,117 @@ def get_pandas_dataframe(self, data_size, summary=None):
         elif self.type == 'distribcell':
             level_df = None
 
-            if isinstance(summary, Summary):
-                # Attempt to import the OpenCG package
-                try:
-                    import opencg
-                except ImportError:
-                    msg = 'The OpenCG package must be installed ' \
-                          'to use a Summary for distribcell dataframes'
-                    raise ImportError(msg)
-
-                # Extract the OpenCG geometry from the Summary
-                opencg_geometry = summary.opencg_geometry
-                openmc_geometry = summary.openmc_geometry
-
-                # Use OpenCG to compute the number of regions
-                opencg_geometry.initialize_cell_offsets()
-                num_regions = opencg_geometry.num_regions
-
-                # Initialize a dictionary mapping OpenMC distribcell
-                # offsets to OpenCG LocalCoords linked lists
-                offsets_to_coords = {}
-
-                for offset, path in enumerate(self.distribcell_paths):
-                    region = opencg_geometry.get_region_from_path(path)
-                    coords = opencg_geometry.find_region(region)
-                    offsets_to_coords[offset] = coords
-
-                # Each distribcell offset is a DataFrame bin
-                # Unravel the paths into DataFrame columns
-                num_offsets = len(offsets_to_coords)
-
-                # Initialize termination condition for while loop
-                levels_remain = True
-                counter = 0
+            # Create Pandas Multi-index columns for each level in CSG tree
+            if distribcell_paths:
+
+                # Distribcell paths require linked metadata from the Summary
+                if self.distribcell_paths is None:
+                    msg = 'Unable to construct distribcell paths since '
```

Comment:
Yes, this is why Travis failed. This is what I get for relying too much on the pampering of PyCharm's IDE - I can't write a simple block of code in emacs anymore! This is fixed and hopefully the tests will pass.